### DISCUSSION TITLE
scrollbar breaks javascript function

### DISCUSSION THREAD

**Discussion by Hautzii** [buzinas/simple-scrollbar#67] [Q&A]
I have a function that scrolls down the page when you click on a button:
`$(".topBar-Title").click(function () {
        $("html, body").animate({
            scrollTop: $("#firstDiv").offset().top
        }, 1000);
    });`
When I add ss-container to the body tag these functions break. 
Does anyone have any idea why ?

**Comment by buzinas** [comment]
Because SimpleScrollbar changes your DOM structure. In such case, you'd need to do something like:

```js
$(".topBar-Title").click(function () {
  $("html, body").animate({ scrollTop: $("#firstDiv .ss-content").offset().top }, 1000);
});
```

**Reply by Hautzii** [reply]
Just tried it and it didn't work, I even tried to add a ss-container class to the different divs.
